AxonDeepSeg/testing/launch_performance_metrics.py

```diff
@@ -1,37 +1,30 @@
-from skimage import measure
-from skimage.measure import regionprops
+"""
+#TODO: Docstring
```

Comment:
We probably shouldn't dirty the code with to-do lists; that's what Github issues are for